src/templates/accountNav.html

```diff
@@ -0,0 +1,32 @@
+<ul class="list pa0 ml3" style="position: sticky; top: 0; align-self: flex-start;">
```

Comment:
I'm not seeing anywhere where this template is used. Is this a leftover?

Reply:
oh ya it is. gonna delete it.